app/views/messages/_messages_sidebar.html.erb

```diff
@@ -1,19 +1,21 @@
 <ul class="messages-sidebar" ng-class="{'focused': selectedMessage !== null}">
-  <li class="borderless">
-    <select name="messageFilter"
-            id="messageFilter"
-            ng-model="messageFilter"
-            ng-change="resetMessages()"
-            ng-options="filter as filter.title for filter in filterOptions">
-    </select>
-  </li>
+  <% unless espionage === 'true' %>
```

Comment:
In order to reuse the same partials between Messages and Espionage, they now pass a variable to the Sidebar which decides whether it's Espionage or not. If Espionage, we do not display the links for creating a new message, or for setting filters based on "Things I have/haven't responded to", since I've responded to nothing.